core/src/stored/stored.cc

```diff
@@ -96,8 +96,9 @@ static void usage()
         "        -dt         print timestamp in debug output\n"
         "        -f          run in foreground (for debugging)\n"
         "        -g <group>  run as group <group>\n"
+        "        -k          keep going despite I/O errors\n"
         "        -m          print kaboom output (for debugging)\n"
-        "        -p          proceed despite I/O errors\n"
+        "        -p <path>   use <path> as destination when creating pidfile\n"
```

Comment:
The proceed despite I/O errors functionality should still exist, but we need to use a different character. (-i like "ignore I/O Errors" maybe?)

Reply:
If we change the flags, probably we need to update the man pages and the docs also. We can create manpages directly via sphinx, we should check if we want to switch the other manpages also like we have it currently for the baeos-dbcopy manpage.